metecho/api/gh.py

```diff
@@ -147,6 +147,11 @@ def get_cumulus_prefix(**kwargs):
     """
     Expects to be in a local_github_checkout.
     """
+    branch_prefix = kwargs.pop("branch_prefix", None)
+    if branch_prefix:
+        return branch_prefix
+    if settings.BRANCH_PREFIX:
+        return settings.BRANCH_PREFIX
```

Comment:
@wlonk I understand the theoretical value of doing this check here inside `get_cumulus_prefix`, but I might suggest moving those out into the `project_create_branch` job where this is called. A few thoughts:

1. This is no longer actually returning the `cumulus_prefix` specifically.
2. Both of these new branches (either `repository.branch_prefix` or `settings.BRANCH_PREFIX`) would not have to be called inside a `local_github_checkout`, so we could avoid that entirely in those cases.

Reply:
I was having some feelings along these lines. You've convinced me.